README.md

```diff
@@ -1,44 +1,238 @@
 # Vault Integration Pack
+_StackStorm pack integration with HashiCorp Vault_
+
+*Author:* steve.neuharth <steve.neuharth@target.com>
+
+## Maintainers
+Active pack maintainers with review & write repository access and expertise with vault:
+* Jacob Floyd ([@cognifloyd](https://github.com/cognifloyd)) <cognifloyd@gmail.com> Copart
+* Carlos ([@nzlosh](https://github.com/nzlosh))
```

Comment:
For someone looking at the pack, the ideal case is when the most important pack information is in the beginning, - it's pack description/configuration instructions/actions/etc.

Please move `Maintainers` to the footer of the Readme. We really need to follow the same standard/format as in the other packs.